k8s/install/README.md

```diff
@@ -0,0 +1,132 @@
+# Installing Kubernetes on zCompute
```

Comment:
please validate the order of the steps as we talked, the CA certificate step is after the `terraform apply`

Reply:
done